### PR TITLE
Feature/xxx task risk calculation migration

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
After we changed storage of RiskCalculatinResult handling old and new Json format is required
Added an initializer to read both formats
To verify it works added some unit-tests

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4615

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class RiskCalculationResultTests: XCTestCase {
+
+	func testGIVEN_NewFormatRiskCalculationResultWithoutRiskLeevlPerDay_WHEN_ParseJson_THEN_RiskCalculationResultGetsCreated() {
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift:10]:
```suggestion
	func testGIVEN_NewFormatRiskCalculationResultWithoutRiskLevelPerDay_WHEN_ParseJson_THEN_RiskCalculationResultGetsCreated() {
```

File: src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+		// WHEN
+		let riskCalculationResult = try? JSONDecoder().decode(RiskCalculationResult.self, from: XCTUnwrap(newFormattedData))
+
+		// THEN
+		XCTAssertNotNil(riskCalculationResult)
+		XCTAssertEqual(riskCalculationResult?.riskLevelPerDate.count, 0)
+	}
+
+	func testGIVEN_NewFormatRiskCalculationResultWithThreeRiskLeevlPerDay_WHEN_ParseJson_THEN_RiskCalculationResultGetsCreated() {
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift:23]:
```suggestion
	func testGIVEN_NewFormatRiskCalculationResultWithThreeRiskLevelPerDay_WHEN_ParseJson_THEN_RiskCalculationResultGetsCreated() {
```

**Review by pascal-brause — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift (hunk 2 above)
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift:23]: I would prefer a proper documentation over the concatenated implementation code as function name.

File: src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		// GIVEN
+		let newFormattedData = "{\"minimumDistinctEncountersWithHighRisk\":0,\"mostRecentDateWithLowRisk\":632530800,\"riskLevelPerDate\":[632530800,1,632444400,1,632358000,1],\"numberOfDaysWithLowRisk\":3,\"calculationDate\":632756849.53607297,\"numberOfDaysWithHighRisk\":0,\"minimumDistinctEncountersWithLowRisk\":3,\"riskLevel\":1}"
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift:25]: Maybe try multi line strings next time. this increases readability and we don't have to escape quotes.

```
		let newFormattedData = """
		json...
		"""
```

File: src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift (hunk 2 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift]: I added some comments

File: src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift (hunk 3 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift:25]: good point, will change that

**Review by marcussc — APPROVED** [review]

**Pull request merged**